hack/build/ci/snyk-image-import.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -x
+
+readonly snyk_org_id=${1}
+readonly snyk_int_id=${2}
+readonly snyk_api_token=${3}
+readonly image=${4}
+
+response=$(curl --include \
```

Comment:
we could also replace this with a dedicated action as suggested by @luhi-DT : https://github.com/marketplace/actions/http-request-action